.github/workflows/frontend-ci.yml

```diff
@@ -336,6 +336,41 @@ jobs:
         if: ${{ env.is-pull-request == 'false' }}
         run: yarn lint --max-parallel=$NX_MAX_PARALLEL
 
+  License_check:
+    name: License check
```

Comment:
Consider a more explicit naming on this, since we already have "checks" which adds and checks for our own license.

Reply:
How about "Code Dependencies License Check"? :)

Reply:
or "Production Dependencies License Check"

Reply:
Fine by me. 
`Allowed licenses check` could also be an option, whatever you choose just that we easily know what's it all about and don't confuse it with our existing license check :)